linux_os/guide/system/auditing/auditd_configure_rules/directory_access_var_log_audit/rule.yml

```diff
@@ -0,0 +1,36 @@
+documentation_complete: true
+
+title: 'Record Access Events to Audit Log directory'
+
+description: |-
+    The audit system should collect access events to read audit log directory.
+    The following audit rule will assure that access to audit log directory are
+    collected.
+    <pre>-a always,exit -F dir=/var/log/audit/ -F perm=r -F auid>=1000 -F auid!=unset -F key=access-audit-trail</pre>
+    If the <tt>auditd</tt> daemon is configured to use the <tt>augenrules</tt>
+    program to read audit rules during daemon startup (the default), add the
+    rule to a file with suffix <tt>.rules</tt> in the directory
+    <tt>/etc/audit/rules.d</tt>.
+    If the <tt>auditd</tt> daemon is configured to use the <tt>auditctl</tt>
+    utility to read audit rules during daemon startup, add the rule to
+    <tt>/etc/audit/audit.rules</tt> file.
+
+rationale: |-
+    Attempts to read the logs should be recorded, suspicious access to audit log files could be an indicator of malicious activity on a system.
+    Auditing these events could serve as evidence of potential system compromise.'
+
+references:
+    ospp@rhel7: FAU_GEN.1.1.c
+
+severity: unknown
+
+ocil_clause: "no line is returned"
+
+ocil: |-
+    Run the following command to check the mode of the system audit logs:
+    <pre>$ ls -ld /var/log/audit</pre>
```

Comment:
This check for mode of `/var/log/audit` is onlyu present in ocil. Is that intended? Or should it go away?

Reply:
I have removed it. Thanks.